qubes/devices.py

```diff
@@ -351,24 +353,19 @@ def assignments(self, persistent=None):
                 self._bus))
             if persistent is True:
                 # don't break app.save()
-                return self._set
+                return list(self._set)
```

Comment:
Could these be sorted?

Reply:
I know the line looks a bit silly, because it looks like we are converting a set (i.e., randomly sorted collection) to a list. However, `self._set` is a PersistentCollection, which is backed by dict. So, IIUC, PersistentCollection keeps the order of the items.

Yeah, we could rename `self._set` to some more descriptive. But it is a set (it cannot contain any item twice), just not an ordinary Python set.